### ISSUE TITLE
Remove unused manifest permissions

### ISSUE THREAD

**Issue by beaufortfrancois** [kzahel/jstorrent-legacy-app#36]
It looks like these permissions are not used: `clipboardWrite, idle, pushMessaging, identity, https://www.googleapis.com/`

**Comment by kzahel** [comment]
You're right, they aren't (though I wanted to use identity and make the app a free trial, but requesting only the openid scope requires user interaction, which I don't like

I know there's another type of free trial for the store, but I don't understand how that one works. Maybe I should look into that.

**Comment by kzahel** [comment]
I removed some of the permissions. Interestingly this seems to have resulted in the permissions prompt ("app disabled because it requested new permissions") for some users.
